packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -136,7 +140,7 @@ export interface ServerlessClusterProps {
   /**
    * Security group.
    *
-   * @default - a new security group is created.
+   * @default - a new security group is created if a vpc is provided.
```

Comment:
What happens if `vpc` is not provided? Probably we need to move the description of that from the `vpc` property documentation to here.

Reply:
👍🏻